### ISSUE TITLE
Change default values of Network visualization

### ISSUE THREAD

**Issue by FBumann** [flixOpt/flixOpt#66]
To make the network visualization better to work with ihr, a more compact view would be nice

**Comment by FBumann** [comment]
Improve path handling:

```
def visualize_network(node_infos: dict,
                      edge_infos: dict,
                      path: Optional[Union[str, pathlib.Path]] = None,
                      controls: Union[bool, List[Literal[
                          'nodes', 'edges', 'layout', 'interaction', 'manipulation',
                          'physics', 'selection', 'renderer']]] = True,
                      show: bool = True
                      ) -> Optional['pyvis.network.Network']:
    """
    Visualizes the network structure of a FlowSystem using PyVis, using info-dictionaries.

    Parameters:
    - path (Union[bool, str, pathlib.Path], default='results/network.html'):
      Path to save the HTML visualization.
        - `False`: Visualization is created but not saved.
        - `str` or `Path`: Specifies file path (default: 'results/network.html').

    - controls (Union[bool, List[str]], default=True):
      UI controls to add to the visualization.
        - `True`: Enables all available controls.
        - `List`: Specify controls, e.g., ['nodes', 'layout'].
        - Options: 'nodes', 'edges', 'layout', 'interaction', 'manipulation', 'physics', 'selection', 'renderer'.
        You can play with these and generate a Dictionary from it that can be applied to the network returned by this function.
        network.set_options()
        https://pyvis.readthedocs.io/en/latest/tutorial.html

    - show (bool, default=True):
      Whether to open the visualization in the web browser.
      The calculation must be saved to show it. If no path is given, it defaults to 'network.html'.

    Returns:
    - Optional[pyvis.network.Network]: The `Network` instance representing the visualization, or `None` if `pyvis` is not installed.

    Usage:
    - Visualize and open the network with default options:
      >>> self.visualize_network()

    - Save the visualization without opening:
      >>> self.visualize_network(show=False)

    - Visualize with custom controls and path:
      >>> self.visualize_network(path='output/custom_network.html', controls=['nodes', 'layout'])

    Notes:
    - This function requires `pyvis`. If not installed, the function prints a warning and returns `None`.
    - Nodes are styled based on type (e.g., circles for buses, boxes for components) and annotated with node information.
    """
    try:
        from pyvis.network import Network
    except ImportError:
        print("Please install pyvis to visualize the network: 'pip install pyvis'")
        return None

    net = Network(directed=True, height='100%' if controls is False else '800px', font_color="white")

    for id, node in node_infos.items():
        net.add_node(id,
                     label=node['label'],
                     shape={'Bus': 'circle', 'Component': 'box'}[node['class']],
                     color={'Bus': '#393E46', 'Component': '#00ADB5'}[node['class']],
                     title=node['infos'].replace(')', '\n)'),
                     font={'size': 14})

    for id, edge in edge_infos.items():
        net.add_edge(edge['start'],
                     edge['end'],
                     label=edge['label'],
                     title=edge['infos'].replace(')', '\n)'),
                     font={"color": "#4D4D4D", "size": 14},
                     color="#222831")

    # Enhanced physics settings
    net.barnes_hut(central_gravity=0.8, spring_length=50, spring_strength=0.2)

    if controls:
        net.show_buttons(filter_=controls)  # Adds UI buttons to control physics settings
    if not show and not path:
        return net

    if path:
        path = pathlib.Path(path).resolve().as_posix() if isinstance(path, str) else path.resolve().as_posix()
        net.write_html(path)
    else:
        path = pathlib.Path('network.html').resolve().as_posix()
    if show:
        net.write_html(path)
        try:
            import webbrowser
            webbrowser.open(f'file://{path}', 2)
        except Exception:
            logger.warning(f'Showing the network in the Browser went wrong. Open it manually. '
                           f'Its saved under {path}')

    return net
```

**Comment by FBumann** [comment]
#116

**Comment by FBumann** [comment]
Closed with #117
